src/manifest.json

```diff
@@ -99,6 +99,12 @@
       "matches": ["*://*/*"],
       "run_at": "document_end",
       "js": ["js/contentScript.bundle.js"]
```

Comment:
Should we maybe rename the `contentScript.bundle.js` to `contentScriptOnEnd.bundle.js` for consistency?

Reply:
+1